test/molecule-role/molecule/default/create.yml

```diff
@@ -28,7 +34,7 @@
         to_port: 0
         cidr_ip: '0.0.0.0/0'
 
-    keypair_name: molecule_key
+    keypair_name: "molecule_key_{{ molecule_run_id }}"
```

Comment:
why do we need to override the key name?

Reply:
tool drops one during the run, if not present locally

Option: provide that key,  than can be reused

Reply:
i prefer the current solution. I'd rather not share keys. The key is witness of the fact the we are authorized for only for the current molecule_run_id